### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -23266,7 +23287,16 @@ termine->determine
 termined->terminated
 terminte->terminate
 termostat->thermostat
+termperatue->temperature
+termperatues->temperatures
 termperature->temperature
+termperatures->temperatures
+termplate->template
+termplates->templates
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also add `templated` as another typo with the same base misspelling.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -23406,6 +23436,8 @@ thiks->thinks
 thimngs->things
 thinigs->things
 thinkabel->thinkable
+thinkg->think, thing, things,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Should this suggest `things`, it seems unlikely?

**Comment by pabs3** [comment]
All the submissions I make have either been seen in the wild or are
variants of misspellings seen in the wild. I record misspellings I see
in bug reports and emails and wiki pages and news articles and more and
then search on Debian Code Search for variants of them.

There is one location where "sucesscient" is present:

https://codesearch.debian.net/search?q=sucesscient&literal=1

There is one location where "thinkg" is a "things" misspelling:

https://codesearch.debian.net/search?literal=1&q=thinkg&page=2

I'll fix up the other suggestions later today.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by pabs3** [comment]
I've made the requested changes and the PR is ready for merging.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Pull request merged**